docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -429,6 +429,15 @@ relation to port exhaustion.
 - `BLACKLISTED_URIS`: **\<empty\>**: If non-empty, list of POSIX regex patterns matching
    OpenID URI's to block.
 
+## OAuth2 Client (`oauth2_client`)
+
+- `REGISTER_EMAIL_CONFIRM`: *[service]* **REGISTER\_EMAIL\_CONFIRM**: Set this to enable or disable email confirmation of OAuth2 auto-registration. (Overwrites the REGISTER\_EMAIL\_CONFIRM setting of the `[service]` section)
+- `OPENID_CONNECT_SCOPES`: **\<empty\>**: List of additional openid connect scopes. (`openid` is implicitly added)
+- `ENABLE_AUTO_REGISTRATION`: **false**: Enable this to allow auto-registration for oauth2 authentication.
+- `USERNAME`: **nickname**: The source of the username for new oauth2 accounts: userid, nickname, email (username part).
+- `UPDATE_AVATAR`: **false**: Set this to update user avatar if available from the oauth2 provider.
+- `ACCOUNT_LINKING`: **disabled**: How to handle if an account / email already exists: disabled / login / auto.
```

Comment:
I would like to copy the description from ini file here and add a warning for `auto` because it may automatically link the two accounts only because the two account have same names. It has some security risks.

Reply:
It mostly make sense if you use email to match + require email validation

Reply:
Yeah, this should also be added to the docs.

Reply:
I agree, should we add the long description for the `USERNAME` setting as well?

Reply:
@mgjm I think so.

Reply:
Added thanks to @kvaster.